### ISSUE TITLE
Bug when clickong cancel with version 1.6.3

### ISSUE THREAD

**Issue by Bidou44** [firescript/nativescript-contacts#99]
After updating to version 1.6.3 and releasing on prod, I can confirm that the bug https://github.com/firescript/nativescript-contacts/issues/75 is now fixed! Thanks again @dlcole @firescript 

Unfortunately, I also noticed that a new issue raised up. Looking at the statistics it seems that 1-2% of the users have an issue that makes the application crash. Actually it's quite simple to reproduce, you have to open the contact plugin and click on cancel. to close it Usually it's working, but sometime it's not. I was able to reproduce it on some phone and some other not.

On average, I would say that I have to open/close the contact plugin more or less 5 times before the application crashes (sometimes it crashes after the first time, sometimes you have to try 20 times and as said, on some devices, its never crashing...)

Any idea what the problem could be? In my opinion, it's new with the version 1.6.3

**Comment by Bidou44** [comment]
After a few days of use on production, the statistics of the users having a crash increased to more than 2% (only iOS).
I had to do a rollback. I cannot recommend using the latest version 1.6.3 (even if it solves an issue) as it is more buggy that the previous, unfortunately.

**Comment by Bidou44** [comment]
@firescript Further info: Doing a rollback didn't help (App is always crashing for 2% if my user on prod)  so I think that this bug is caused by Nativescript 8 (while moving to nativescript-contacts 1.6.3 I also upgraded to Nativescript 8). Maybe it's worth looking at this PR: https://github.com/firescript/nativescript-contacts/pull/97
What do you think?

**Comment by firescript** [comment]
We could merge that, and bump npm again. Would that solve your issue?

**Comment by Bidou44** [comment]
I didn't check the content/quality of this PR, but if the migration was correctly done, then yes if would really help!

**Comment by Bidou44** [comment]
@firescript Any news on this? Would it be possible to merge the PR in a branch and publish a beta version? I can then do some tests to check if is solves the problem.

**Comment by ragu89** [comment]
Now that the [pull request 97](https://github.com/firescript/nativescript-contacts/pull/97) is merged , waiting for the next release. 
Is it planned Ryan?

**Comment by Bidou44** [comment]
@firescript Yes it would be good if you could release a new version at least a beta so that I can make some tests...
